### PR TITLE
Cone and Streamtube plot doc update

### LINKED ISSUE #380: [RCE] Add new 3D Charts Tutorials
- [x] Cone Plots
- [x] Streamtubes

general tutorial advice:
- keep the tutorial simple :)
- use Plotly colors for styling (when styling the chart)
- keep directions short and direct 
- use a real dataset
- use Plotly colors for styling (when styling the chart)
- short gifs and images are great 🎉

### PR DESCRIPTION
**Pull request by priyatharsan**
closes #380 
and duplicate of https://github.com/plotly/plotly.github.io/pull/371

### CODE REVIEW

File: _posts/3d/2018-11-05-how-to-create-a-streamtube-plot.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+layout: tutorial-single_layout
+title: 3D Streamtube Plots
+subtitle: A streamtube is a tubular region surrounded by streamlines that form a closed loop. It's a continuous version of a streamtube plot (3-D quiver plot) and can provide insight into flow data from natural systems.
+permalink: /make-a-streamtube-plot/
+imageurl: ../static/images/create-streamtube-plot/3d-streamtube-thumbnail.png
```
Comment (cldougl) [inline _posts/3d/2018-11-05-how-to-create-a-streamtube-plot.md]: can the thumbnail be an image of the branded chart

File: _posts/3d/2018-11-05-how-to-create-a-cone-plot.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
+---
+layout: tutorial-single_layout
+title: 3D Cone Plots
+subtitle: Create 3D Cone Vortex using cone plots.
```
Comment (cldougl) [inline _posts/3d/2018-11-05-how-to-create-a-cone-plot.md]: Create a 3D vortex using cone plots.

File: _posts/3d/2018-11-05-how-to-create-a-cone-plot.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+ - title: Add Your Data to Plotly
+   sub-steps:
+    - copy: "Head to Plotly’s [Chart Studio](https://plot.ly/create/) and add your data. You have the option of typing directly in the grid, uploading your file, or entering a URL of an online dataset. Plotly accepts .xls, .xlsx, or .csv files. For more information on how to enter your data, see [this](https://help.plot.ly/add-data-to-the-plotly-grid/) tutorial."
+
+ - title: Create a Chart
+   sub-steps:
+    - copy: "After adding data, go to the 'Traces' section under the 'Structure' menu on the left-hand side. Choose the 'Type' of trace, then choose 'Cone' under '3D' chart type."
+      img: "![Chart type cone](../static/images/create-cone-plot/chart-type-cone.png)"
+    - copy: Next, fill the 'X', 'Y', 'Z', 'U', 'V' and 'W' values from the dropdown to create the plot. This will create a raw trace for the vortex data, as seen below."
```
Comment (cldougl) [inline _posts/3d/2018-11-05-how-to-create-a-cone-plot.md]: @priyatharsan  Can you write a bit more about what each of these values represents.

File: _posts/3d/2018-11-05-how-to-create-a-cone-plot.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+      img: "![Axes data](../static/images/create-cone-plot/select-axes-data.gif)"
+
+ - title: Style Trace
+   sub-steps:
+    - copy: "The 'Style' menu displays many options to modify characteristics of the overall chart layout or the individual traces. To see more options about styling the chart, visit the [style and layout](https://help.plot.ly/tutorials/#layout) section of the Chart Studio documentation."
```
Comment (cldougl) [inline _posts/3d/2018-11-05-how-to-create-a-cone-plot.md]: Overall, I think we should go through the style options linearly -> general / traces / any others in order

File: _posts/3d/2018-11-05-how-to-create-a-streamtube-plot.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+state: active
+tags: graph3d
+order: 4
+meta_description: A tutorial on how to make a streamtube plot in Chart Studio.
+popularity: featured
+carouselimageurl: ../static/images/create-streamtube-plot/3d-streamtube-thumbnail.png
+actioncall: How to make 3D streamtube plots in Plotly's Chart Studio.
+actioncall-url: https://plot.ly/create/?fid=plotly2_demo:428
```
Comment (cldougl) [inline _posts/3d/2018-11-05-how-to-create-a-streamtube-plot.md]: the title isn't quite gramatically correct-
should be Streamtubes representing the ...
or Streamtubes of the...

File: _posts/3d/2018-11-05-how-to-create-a-streamtube-plot.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+ - title: Create a Chart
+   sub-steps:
+    - copy: "After adding data, go to the 'Traces' section under the 'Structure' menu on the left-hand side. Choose the 'Type' of trace, then choose 'Streamtube' under '3D' chart type."
+      img: "![Chart type streamtube](../static/images/create-streamtube-plot/chart-type-streamtube.png)"
+    - copy: Next, fill the 'X', 'Y', 'Z', 'U', 'V', 'W', 'X Start', 'Y start', and 'Z Start' values from the dropdown to create the plot. This will create a raw trace for the str data, as seen below."
+      img: "![Axes data](../static/images/create-streamtube-plot/select-axes-data.png)"
+
+ - title: Style Trace
+   sub-steps:
+    - copy: "The 'Style' menu displays many options to modify characteristics of the overall chart layout or the individual traces. To see more options about styling the chart, visit the [style and layout](https://help.plot.ly/tutorials/#layout) section of the Chart Studio documentation."
```
Comment (cldougl) [inline _posts/3d/2018-11-05-how-to-create-a-streamtube-plot.md]: general -> style

File: _posts/3d/2018-11-05-how-to-create-a-streamtube-plot.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+
+ - title: Add Your Data to Plotly
+   sub-steps:
+    - copy: "Head to Plotly’s [Chart Studio](https://plot.ly/create/) and add your data. You have the option of typing directly in the grid, uploading your file, or entering a URL of an online dataset. Plotly accepts .xls, .xlsx, or .csv files. For more information on how to enter your data, see [this](https://help.plot.ly/add-data-to-the-plotly-grid/) tutorial."
+
+ - title: Create a Chart
+   sub-steps:
+    - copy: "After adding data, go to the 'Traces' section under the 'Structure' menu on the left-hand side. Choose the 'Type' of trace, then choose 'Streamtube' under '3D' chart type."
+      img: "![Chart type streamtube](../static/images/create-streamtube-plot/chart-type-streamtube.png)"
+    - copy: Next, fill the 'X', 'Y', 'Z', 'U', 'V', 'W', 'X Start', 'Y start', and 'Z Start' values from the dropdown to create the plot. This will create a raw trace for the str data, as seen below."
```
Comment (cldougl) [inline _posts/3d/2018-11-05-how-to-create-a-streamtube-plot.md]: can you describe with some detail what x/y/z/u/v/w represent

File: _posts/3d/2018-11-05-how-to-create-a-cone-plot.md (hunk 2 above)
Comment (priyatharsan) [reply _posts/3d/2018-11-05-how-to-create-a-cone-plot.md]: 👍

File: _posts/3d/2018-11-05-how-to-create-a-cone-plot.md (hunk 3 above)
Comment (priyatharsan) [reply _posts/3d/2018-11-05-how-to-create-a-cone-plot.md]: 👍

File: _posts/3d/2018-11-05-how-to-create-a-cone-plot.md (hunk 4 above)
Comment (priyatharsan) [reply _posts/3d/2018-11-05-how-to-create-a-cone-plot.md]: 👍

File: _posts/3d/2018-11-05-how-to-create-a-streamtube-plot.md (hunk 1 above)
Comment (priyatharsan) [reply _posts/3d/2018-11-05-how-to-create-a-streamtube-plot.md]: 👍

File: _posts/3d/2018-11-05-how-to-create-a-streamtube-plot.md (hunk 5 above)
Comment (priyatharsan) [reply _posts/3d/2018-11-05-how-to-create-a-streamtube-plot.md]: Makes sense. I changed 'streamtube plots' to' streamtubes'. What about the title should it be '3D Streamtube Plots' or just '3D Streamtube'

File: _posts/3d/2018-11-05-how-to-create-a-streamtube-plot.md (hunk 7 above)
Comment (priyatharsan) [reply _posts/3d/2018-11-05-how-to-create-a-streamtube-plot.md]: 👍

File: _posts/3d/2018-11-05-how-to-create-a-streamtube-plot.md (hunk 6 above)
Comment (priyatharsan) [reply _posts/3d/2018-11-05-how-to-create-a-streamtube-plot.md]: 👍

File: _posts/3d/2018-11-05-how-to-create-a-streamtube-plot.md [hunk 8]
```diff
@@ -1,16 +1,16 @@
 ---
 layout: tutorial-single_layout
 title: 3D Streamtube Plots
-subtitle: A streamtube is a tubular region surrounded by streamlines that form a closed loop. It's a continuous version of a streamtube plot (3-D quiver plot) and can provide insight into flow data from natural systems.
+subtitle: Use 3D streamtubes to visualize flow in a vector field in 3-dimensions.
```
Comment (cldougl) [inline _posts/3d/2018-11-05-how-to-create-a-streamtube-plot.md]: the subtitle was ok! I like the descriptive version :) 
> A streamtube is a tubular region surrounded by streamlines that form a closed loop. It's a continuous version of a streamtube plot (3-D quiver plot) and can provide insight into flow data from natural systems.

just change 3-D to 3D

File: _posts/3d/2018-11-05-how-to-create-a-streamtube-plot.md (hunk 5 above)
Comment (cldougl) [reply _posts/3d/2018-11-05-how-to-create-a-streamtube-plot.md]: I think 3D Streamtube Plots is good 👍

File: _posts/3d/2018-11-05-how-to-create-a-streamtube-plot.md (hunk 8 above)
Comment (priyatharsan) [reply _posts/3d/2018-11-05-how-to-create-a-streamtube-plot.md]: 👍

**Pull request merged**